File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        for (var i = 0; i < nums.length; i++) {
    
    for (var j = i; j < nums.length - 1; j++) {
      if (nums[i] + nums[j + 1] == target) {
        return [ i,j+1];
      }
    }
  }
  
}
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = (2 * len(nums))
        ans:list = [0] * length

        for i in range(len(nums)):
            ans[i] = nums[i]
            ans[i+len(nums)] = nums[i]
        return ans
        